chapter10/ajd/exercise2.pl

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use Benchmark qw( :all );
+
+sub naive {
+  chdir;
+  my @sorted = sort { -s $a <=> -s $b } glob '*';
+  # Explicit return value to stop the array being returned
+  # Turns out Benchmark goes off and tests all the scripts that this returns
```

Comment:
I found that after it ran for ages. Cancelled the first couple, then left it going to see what happened. Ended up with a huge performance report of all the scripts. I probably just needed some more parentheses.